### ISSUE TITLE
Package control crashes Sublime Text 3

### ISSUE THREAD

**Issue by jago86** [wbond/package_control#988]
Guys, I tried to install PAckage Control on sublime text 3 in ubuntu 14.04 and not just don't install, but sublime crashes!! I reopened and crashes again. Sublime is now unusable. Any ideas?

**Comment by littleCoderVA** [comment]
I have the similar issue too. Mine is I enter the command into subl3 and enter the whole thing just freeze. I have to go inside Installed Packages folder and remove both 0_package_control_loader.sublime-package and Package Control.sublime-package then sublime starts working again, but without the package control

**Comment by joker-b** [comment]
Having the same issue, Package Control freezes Sublime in ubunto 14.10 - clobbering ~/.config/sublime-text-3/ will restore Sublime at the expense of needing to re-enter the license...

**Comment by littleCoderVA** [comment]
Hi joker-b, after you remove the sublime-text-3 folder and re-enter the license can you have the package control working? I have tried so many ways but as long as 0_package_control_loader.sublime-package and Package Control.sublime-package get into Installed Packages folder sublime freeze up.

**Comment by joker-b** [comment]
sadly, no. I just toast the .config dir to restore Sublime to "factory-fresh" state.

Package Control still fails

**Comment by mystik3eb** [comment]
I'm having very similar issues. I just installed Linux Mint and have been trying to install Package Control in Sublime Text 3, but as soon as I try, Sublime is unusable, the menus can't be selected, the window can't be resized (rather it tries, but the inner window doesn't resize with the rest of the window), text can't be entered, nothing. It starts working again as soon as I remove the two packages from Installed Packages. It doesn't matter whether I drag the package into the folder, use the Python script in the Sublime Console, or clone with git, behavior is identical regardless.

**Comment by lamialily** [comment]
Can confirm. Fresh install of build 3083 of Sublime Text 3 and latest of Package Control pulled from the site over the last 32 hours, under Antergos (Arch fork) Linux i386 -- Package Control freezes the application as soon as it is loaded, based on the brief amount of time I have to open a console. No useful information was presented by putting the debug flag in my preferences. And here I thought it was a fluke and didn't think about reporting it until my boyfriend pointed me to this issue!

**Comment by michalpristas** [comment]
tried installing with the script as well as wget into ~/.config/sublime-text-3/Installed Packages
sublime freezes after start

using 	Ubuntu 14.04.3 LTS
Release:	14.04

**Comment by mrnovvad** [comment]
Can confirm too. Just installed Arch Linux and tried to install Package Control. Now Sublime freezes after start

**Comment by wbond** [comment]
Anyone not using Linux?

**Comment by wbond** [comment]
I just tested installing Package Control on Ubuntu 15.04 x64 without issues.

What architecture are all of you using? x86_64 or i686?

**Comment by Jancis** [comment]
Hangs with 15.04 x64 here. Probably some package is missing.

**Comment by v1kku** [comment]
I'm on Ubuntu 14.04 x64. Having the same issue.

**Comment by littleCoderVA** [comment]
Same, x64

**Comment by wbond** [comment]
Were any of you successfully running Package Control 3.0.1?

**Comment by OOKAMISHURA** [comment]
having the same issues with package controll , sublime freezez , freshly installed ubuntu 14.04 x64

**Comment by wbond** [comment]
Could everyone running Linux x64 open a terminal and try the following:

 - `cd ~/.config/sublime-text-3/Packages/`
 - See if a folder named `ssl-linux` exists
 - Try executing `ldd ssl-linux/st3_linux_x64/libssl-1.0.0/_ssl.cpython-33m.so`

**Comment by petuomin** [comment]
No `ssl-linux`. (Ubuntu 15.04, x64).

**Comment by wbond** [comment]
@petuomin Do you have a folder named `bz2`?

**Comment by petuomin** [comment]
Nope, only folder named `User` there. And that is empty too. This is a fresh install.

**Comment by wbond** [comment]
If anyone wants to hop on freenode to the `##sublimetext` channel, we can try some faster iterations of checking things out. There is a web client at http://webchat.freenode.net/?channels=##sublimetext.

**Comment by wbond** [comment]
@petuomin Can you try executing the following?

```bash
unzip -l ~/.config/sublime-text-3/Installed\ Packages/0_package_control_loader.sublime-package
```

**Comment by sdilipsam** [comment]
Same issue, 
Running Ubuntu Mate 
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.1 LTS
Release:	14.04
Codename:	trusty
Linux dilip-Desktop 3.13.0-39-generic #66-Ubuntu SMP Tue Oct 28 13:30:27 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux

and ST3 Build 3083

**Comment by sdilipsam** [comment]
@petuomin 

I executed the script and this is what I got

  Length      Date    Time    Name
---------  ---------- -----   ----
      175  2015-09-15 18:50   dependency-metadata.json
     1876  2015-09-15 18:50   00-package_control.py
---------                     -------
     2051                     2 files

**Comment by petuomin** [comment]
Exactly same for me, except for the times.

**Comment by wbond** [comment]
I am installing a fresh copy of 14.04 x64 to try and reproduce.

**Comment by wbond** [comment]
I just reproduced the issue on a clean install of ST 3083 on Ubuntu 14.04 x64. Working on a fix now.

**Comment by sdilipsam** [comment]
added bz2 from an installation in my windows env.
also copied the Package Control.sublime-package and 0_package_control_loader.sublime-package

on restarting sublime, there was a prompt on missing dependencies and to restart the ST3.

Works now!

**Comment by wbond** [comment]
The issue seems to be with `2_bootstrap.py`, digging in more right now.

**Comment by wbond** [comment]
This issue is fixed by 3.1.1. If you delete `Installed Packages/Package Control.sublime-package` and follow the directions at https://packagecontrol.io/installation.

**Comment by unabomber87** [comment]
i followed the instruction and reinstalled package control but the problem is that i can't find install package on the command palette

**Comment by lamialily** [comment]
@Unabomber87 I recommend making a new issue for that. Better bookkeeping that way.

**Comment by OriginalEXE** [comment]
Thanks wbond, I can confirm that removing `Installed Packages/Package Control.sublime-package` and reinstalling package control fixes the issue.

**Comment by lamialily** [comment]
Can confirm. Newest version of Package Control works like a dream.

**Comment by jago86** [comment]
Thanks!! Is working now.

**Comment by akakcolin** [comment]
Having the same issue. Now fixed it, Thanks

**Comment by julesmoretti** [comment]
Same issue here:

```
Package Control: Fetching list of available packages
  Platform: osx-x64
  Sublime Text Version: 3114
  Package Control Version: 3.1.2
Package Control: Download Debug
  URL: https://packagecontrol.io/channel_v3.json
  Timeout: 30
  Resolved IP: 50.116.33.29
  Resolved IPv6: 2600:3c02::f03c:91ff:fedf:6883
Package Control: Urllib Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Found previously exported CA bundle at /Users/jules.moretti/Library/Application Support/Sublime Text 3/Packages/User/Package Control.system-ca-bundle
Package Control: Urllib HTTPS Debug General
  Connecting to packagecontrol.io on port 443
Package Control: Downloading https://packagecontrol.io/channel_v3.json timed out, trying again
Package Control: Urllib HTTPS Debug General
  Connecting to packagecontrol.io on port 443
```

Though it simply seems that there servers are down... because binging is not resolving for the domain...

```
[jules.moretti:...orial/lessons/01-setting-up]$ ping packagecontrol.io                    (master✱) 
PING packagecontrol.io (50.116.33.29): 56 data bytes
Request timeout for icmp_seq 0
Request timeout for icmp_seq 1
Request timeout for icmp_seq 2
Request timeout for icmp_seq 3
Request timeout for icmp_seq 4
```

vs google

```
[jules.moretti:...orial/lessons/01-setting-up]$ ping google.com                           (master✱) 
PING google.com (216.58.194.174): 56 data bytes
64 bytes from 216.58.194.174: icmp_seq=0 ttl=54 time=11.589 ms
64 bytes from 216.58.194.174: icmp_seq=1 ttl=54 time=64.252 ms
```

so... patience it will be on my end :)
